packages/jazz-react/src/index.tsx

```diff
@@ -50,72 +50,75 @@ export function createJazzReactApp<Acc extends Account>({
+          const previousContext = ctx;
 
-        if (previousContext) {
-          previousContext.done();
-        }
-      }
-
-      async function createContext() {
-        const currentContext = await createJazzBrowserContext<Acc>(
-          auth === "guest"
-            ? {
-                peer,
-                storage,
-              }
-            : {
-                AccountSchema,
-                auth,
-                peer,
-                storage,
-              },
-        );
-
-        const logOut = () => {
-          currentContext.logOut();
-          setCtx(undefined);
-          setSessionCount(sessionCount + 1);
-
-          if (process.env.NODE_ENV === "development") {
-            // In development mode we don't return a cleanup function
-            // so we mark the context as done here.
-            currentContext.done();
+          if (previousContext) {
+            previousContext.done();
           }
-        };
-
-        setCtx({
-          ...currentContext,
-          logOut,
-        });
+        }
 
-        return currentContext;
-      }
+        async function createContext() {
+          const currentContext = await createJazzBrowserContext<Acc>(
+            auth === "guest"
+              ? {
+                  peer,
+                  storage,
+                }
+              : {
+                  AccountSchema,
+                  auth,
+                  peer,
+                  storage,
+                },
+          );
+
+          const logOut = () => {
+            currentContext.logOut();
+            setCtx(undefined);
+            setSessionCount(sessionCount + 1);
+
+            if (process.env.NODE_ENV === "development") {
+              // In development mode we don't return a cleanup function
+              // so we mark the context as done here.
+              currentContext.done();
+            }
+          };
+
+          setCtx({
+            ...currentContext,
+            logOut,
+          });
+
+          return currentContext;
+        }
 
-      const promise = createContext();
+        const promise = createContext();
 
-      // In development mode we don't return a cleanup function because otherwise
-      // the double effect execution would mark the context as done immediately.
-      if (process.env.NODE_ENV === "development") {
-        return;
-      }
+        // In development mode we don't return a cleanup function because otherwise
+        // the double effect execution would mark the context as done immediately.
+        if (process.env.NODE_ENV === "development") {
+          return;
+        }
 
-      return () => {
-        void promise.then((context) => context.done());
-      };
-    }, [AccountSchema, auth, peer, storage, sessionCount]);
+        return () => {
+          void promise.then((context) => context.done());
+        };
+      },
+      [AccountSchema, auth, peer, sessionCount].concat(storage as any),
```

Comment:
The core change is here, the rest is formatting.

Reply:
Used `concat` because it already handles the difference between a single value and an array